source/nodejs/adaptivecards/src/card-object.ts

```diff
@@ -64,6 +64,15 @@ export abstract class CardObject extends SerializableObject {
     protected _parent?: CardObject;
     protected _renderedElement?: HTMLElement;
 
+    /**
+     * Check if renderedElement contains query Node
```

Comment:
Since this is going to end up in actual documentation, we need to attempt to write in a public-facing way:
- Checks if this object contains the specified DOM node
- @param node The DOM node to look for
- @returns `true` if the DOM node was found, `false` otherwise